packages/image/src/index.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <div ref="container" class="el-image">
+  <div
+    ref="container"
+    class="el-image"
+    :class="$props['class']"
```

Comment:
Is there another way to bind it?

Reply:
Two way:
1. Make `style` and `class` in props and bind `$props['class']` or `$props.class` to `div`(`class` is js keywords, so cannot bind `class` directly).

2. Make `style` and `class` not in props, bind `$attrs.class` and `$attrs.style` to `div` and filte `style` and `class` in `setup` and bind rest attrs to `img`.

Which is better?

Reply:
This is not making any sense, when you declare `style` and `class` in `props`, simply use `style` and `class` would've been enough, you do not need syntax like `$props['class']` also , accessing a member when you already know it's name you do not need the `[]`, simply using dot operator to reach the member

Reply:
`class` is keyword of Javascript, cannot use directly.